File: JJB/1201/점의_위치_구하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> dot) 
{
    int answer = 0;
    
    int x = dot[0];
    int y = dot[1];
    
    switch((x>0)*2+(y>0))
    {
        case 3:
            answer = 1;
            break;
        case 1:
            answer = 2;
            break;
        case 0:
            answer = 3;
            break;
        default:
            answer = 4;
            break;
    }
    
    return answer;
}

File: JJB/Programmers/231201/점의_위치_구하기.py
def solution(dot):
    answer = 0
    
    x = dot[0]
    y = dot[1]
    
    if y > 0:
        if x > 0:
            answer = 1
        else:
            answer = 2
    else:
        if x < 0:
            answer = 3
        else:
            answer = 4
    
    return answer